src/components/VmDetails/style.css

```diff
@@ -101,3 +101,13 @@
 div.base-card.cell-card {
   flex-basis: auto;
 }
+
+@media only screen and (max-width: 600px) {
+  .grid-column{
+    min-width: 100%;
+  }
+  [class*="col-"]{ /*smaller margins for mobile*/
```

Comment:
I think this should be in #1028 since it is global to layout of all cards, not just NICs card content.

Reply:
Same as https://github.com/oVirt/ovirt-web-ui/pull/1029/files#r294304289

Reply:
I did this because NICs layout is different, with IPv4 and IPv6 being in two columns. However, in narrower screen sizes, we just one column for both v4 and v6. Besides, no other card has this layout.